src/datatype.rs

```diff
@@ -21,6 +22,67 @@ const EXCEL_1900_1904_DIFF: f64 = 1462.;
 #[cfg(feature = "dates")]
 const MS_MULTIPLIER: f64 = 24f64 * 60f64 * 60f64 * 1e+3f64;
 
+/// A struct that combines cell value and style information
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct CellData {
+    /// The cell value
+    pub value: Data,
+    /// The cell style
+    pub style: Option<Style>,
+}
+
+impl CellData {
+    /// Create a new CellData with just a value
+    pub fn new(value: Data) -> Self {
+        Self { value, style: None }
+    }
+
+    /// Create a new CellData with value and style
+    pub fn with_style(value: Data, style: Style) -> Self {
+        Self {
+            value,
+            style: Some(style),
+        }
+    }
+
+    /// Get the cell value
+    pub fn get_value(&self) -> &Data {
+        &self.value
+    }
+
+    /// Get the cell style
+    pub fn get_style(&self) -> Option<&Style> {
+        self.style.as_ref()
+    }
+
+    /// Set the cell style
+    pub fn set_style(&mut self, style: Style) {
+        self.style = Some(style);
+    }
+
+    /// Remove the cell style
+    pub fn remove_style(&mut self) {
+        self.style = None;
+    }
+
+    /// Check if the cell has style information
```

Comment:
Again, an Excel cell will always have style information even if it is implicit for size reasons.